docs/contribute.md

```diff
@@ -21,7 +21,7 @@ Please refer to each project's style and contribution guidelines for submitting
  4. **Push** your work 
  5. Submit a **Pull request** so that we can review your changes
 
-
+<!-- FIXME: What does this title mean? I dont get it :P-->
```

Comment:
It looks like they are instructions to set up the Python virtual environment, could you redact the title to something like that?

Reply:
sorry, i never saw the notification